### PR TITLE
Indicate better how to open kibana from Web Console and create the index patterns

### PR DESCRIPTION
**Pull request by mburke5678**
https://bugzilla.redhat.com/show_bug.cgi?id=1886386

### CODE REVIEW

**Comment by mburke5678** [comment]
@anpingli PTAL

File: modules/cluster-logging-visualizer-indices.adoc [hunk 1]
```diff
@@ -33,11 +33,11 @@ The audit logs are not stored in the internal {product-title} Elasticsearch inst
...
-To define index patterns and create visualizations:
+To define index patterns and create visualizations in Kibana:
 
-. Launch Kibana by clicking the Application Launcher {launch} and select *Logging*.
+. In the {product-title} console, click the Application Launcher {launch} and select *Logging*.
 
-. Create your Kibana index patterns:
+. Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
 ** Regular users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs.
```
Comment (anpingli) [inline modules/cluster-logging-visualizer-indices.adoc:42]: Regular users -> Users.   All users should create  Index Pattern themselves in 4.5.

File: modules/cluster-logging-visualizer-indices.adoc [hunk 2]
```diff
@@ -33,14 +33,14 @@ The audit logs are not stored in the internal {product-title} Elasticsearch inst
...
+. In the {product-title} console, click the Application Launcher {launch} and select *Logging*.
 
-. Create your Kibana index patterns:
+. Create your Kibana index patterns by clicking *Management* -> *Index Patterns* -> *Create index pattern*:
 
-** Regular users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
+** Users must manually create index patterns to see logs for their projects. Users should create a new index pattern named `app` and use the `@timestamp` time field to view their container logs. 
 
-** Admin users need to create index patterns for the *app*, *infra*, and *audit* indices using the `@timestamp` time field. 
+** Admin users must create index patterns for the *app*, *infra*, and *audit* indices using the `@timestamp` time field.
```
Comment (jeana-redhat) [inline modules/cluster-logging-visualizer-indices.adoc:44]: In line 42, "`app`" is formatted with `` I assume to indicate user text entry. In line 44, are "app", "infra", and "audit" also user-provided values? i.e., are users and admins doing the same process here, just with differences in what they create? 

Just struck me as a bit odd that they are formatted in bold instead, but that might be right based on the actual process for all I know! :)

**Review by jeana-redhat — COMMENTED** [review]
One question about the procedure, but if the answer to that is "no" then this is all set :)

**Pull request merged**